docs/book/src/tasks/installation.md

```diff
@@ -24,7 +24,7 @@ Cluster API requires an existing kubernetes cluster accessible via kubectl, choo
 
   ```bash
   kind create cluster --name=clusterapi
-  export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+  kubectl cluster-info --context kind-clusterapi
```

Comment:
We should document that these instructions follow kind >= v0.6

Reply:
Is line 22 sufficient for this?

Reply:
I pushed another commit to remove the duplication and make it clear that it's kind versioning we're talking about.

Reply:
This seems extra, can we remove it?

Reply:
Extra in that it duplicates the output of kind itself?  I think it's helpful to call it out. When working with multiple clusters, this line is definitely needed, at least the last time I ran through the install.